src/services/injection.service.ts

```diff
@@ -19,7 +19,8 @@ import {
  */
 @Injectable()
 export class InjectionService {
-  private _container: ComponentRef<any>;
+  static _container: ViewContainerRef;
```

Comment:
let's rename this to something else (globalContaier, staticContainer) please.

Reply:
Will do....